src/CameraRoll.ts

```diff
@@ -238,3 +240,51 @@ export class CameraRoll {
     return RNCCameraRoll.getPhotoByInternalID(internalID, conversionOption);
   }
 }
+
+const initialState: PhotoIdentifiersPage = {
+  edges: [],
+  page_info: {
+    end_cursor: '',
+    has_next_page: false,
+    start_cursor: '',
+  },
+};
+
+const defaultConfig: GetPhotosParams = {
+  first: 20,
+  groupTypes: 'All',
+};
+
+type UseCameraRollResult = [
+  PhotoIdentifiersPage,
+  (config?: GetPhotosParams) => Promise<void>,
+  (tag: string, options?: SaveToCameraRollOptions) => Promise<void>,
+];
+
+export function useCameraRoll(): UseCameraRollResult {
+  const [photos, setPhotos] = useState<PhotoIdentifiersPage>(initialState);
+
+  async function getPhotos(
+    config: GetPhotosParams = defaultConfig,
+  ): Promise<void> {
+    try {
+      const result = await CameraRoll.getPhotos(config);
+      setPhotos(result);
+    } catch (error) {
+      if (__DEV__) console.log('[useCameraRoll] Error getting photos: ', error);
+    }
+  }
+
+  async function saveToCameraRoll(
```

Comment:
Maybe also rename this to just `save`, sorry for being nitpicky. Otherwise it looks good to me

Reply:
no problem